devolutions-gateway/Cargo.toml

```diff
@@ -88,7 +88,7 @@ dlopen = "0.1"
 dlopen_derive = "0.1"
 
 # Video processing for session recording
-cadeau = { version = "0.3", features = ["dlopen"] }
+cadeau = { path = "../../cadeau/rust/cadeau", features = ["dlopen"] }
```

Comment:
Notice: This will and need to be updated with new release of cadeau

Reply:
I published cadeau v0.4.0